modes/contacto.md

```
@@ -1,26 +1,53 @@
+   - **Interviewer** -- alguien que va a entrevistar al candidato (fecha conocida)
 
-3. **Generar mensaje** con framework de 3 frases:
-   - **Frase 1 (Gancho)**: Algo específico sobre su empresa o reto actual con IA (NO genérico)
-   - **Frase 2 (Prueba)**: Mayor logro cuantificable del candidato relevante para ESE rol (ej: "I built an AI agent handling 90% of customer service at my company before selling it")
-   - **Frase 3 (Propuesta)**: Charla rápida, sin presión ("Would love to chat about [tema específico] for 15 min")
+3. **Seleccionar target primario**: la persona que mas se beneficiaria de que el candidato estuviera alli
 
-4. **Versiones**:
+4. **Generar mensaje** con framework de 3 frases adaptado al tipo de contacto:
+
+   ### Recruiter
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)